Gems/RecastNavigation/Code/Source/EditorComponents/EditorRecastNavigationPhysXProviderComponent.cpp

```diff
@@ -27,7 +27,28 @@ namespace RecastNavigation
                     "[Collects triangle geometry from PhysX scene for navigation mesh within the area defined by a shape component.]")
                     ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
                     ->Attribute(AZ::Edit::Attributes::AppearsInAddComponentMenu, AZ_CRC("Game"))
-                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true);
+                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                    ->Attribute(AZ::Edit::Attributes::Visibility, AZ::Edit::PropertyVisibility::ShowChildrenOnly)
+                ;
+
+                editContext->Class<RecastNavigationPhysXProviderComponentController>(
+                    "RecastNavigationPhysXProviderComponentController", "")
```

Comment:
is "RecastNavigationPhysXProviderComponentController" going to be shown to the user? If it is it'll be more readable with spaces.